ukm-semantics/test/execution.md

```diff
@@ -2,17 +2,31 @@
 
 module UKM-TEST-SYNTAX
     imports INT-SYNTAX
+    imports STRING-SYNTAX
     imports RUST-EXECUTION-TEST-PARSING-SYNTAX
     imports UKM-HOOKS-UKM-SYNTAX
+    imports BYTES-SYNTAX
+
+    syntax UKMTestTypeHolder ::= "ptr_holder" KItem [strict]
+                                | "value_holder" KItem
+                                | "list_ptrs_holder" List
```

Comment:
Would a PtrList work here, and a ValueList below?

Reply:
Added as a TODO. I'll be solving this together with the other issues raised for ukm-semantics/test/execution.md.